src/components/PurchaseList/PurchaseList.js

```diff
@@ -0,0 +1,53 @@
+import styled from 'styled-components/macro'
+import { ReactComponent as DeleteIcon } from '../../images/delete-icon.svg'
+
+export default function PurchaseList({ purchasedFood, onRemoveFood }) {
+  return (
+    <PurchaseCard>
```

Comment:
```suggestion
    <Card>
```

I think only Card is sufficient here as you are in the context of `<PurchaseList>